{"@t":"2025-02-17T08:22:45.6865496Z","@mt":"{StartMessage} [Timing {TimingId}]","StartMessage":"Booting Umbraco 8.6.3.","TimingId":"6cf3b80","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"be245a42-069a-4183-89ec-73b07fdeec78"}
{"@t":"2025-02-17T08:22:45.7418026Z","@mt":"Booting site '{HostingSiteName}', app '{HostingApplicationID}', path '{HostingPhysicalPath}', server '{MachineName}'.","HostingSiteName":"Kingrice","HostingApplicationID":"/LM/W3SVC/2/ROOT","HostingPhysicalPath":"D:\\.Net\\KingriceProduction\\Kingrice\\","MachineName":"DESKTOP-7JFF6SL","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"be245a42-069a-4183-89ec-73b07fdeec78"}
{"@t":"2025-02-17T08:22:46.0847004Z","@mt":"Acquiring.","SourceContext":"Umbraco.Core.Runtime.MainDom","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"be245a42-069a-4183-89ec-73b07fdeec78"}
{"@t":"2025-02-17T08:22:46.0887008Z","@mt":"Acquired.","SourceContext":"Umbraco.Core.Runtime.MainDom","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"be245a42-069a-4183-89ec-73b07fdeec78"}
{"@t":"2025-02-17T08:22:48.6814191Z","@mt":"{StartMessage} [Timing {TimingId}]","StartMessage":"Resolving composer types.","TimingId":"ae3ab19","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"be245a42-069a-4183-89ec-73b07fdeec78"}
{"@t":"2025-02-17T08:22:49.0785596Z","@mt":"{EndMessage} ({Duration}ms) [Timing {TimingId}]","EndMessage":"Resolved.","Duration":396,"TimingId":"ae3ab19","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"be245a42-069a-4183-89ec-73b07fdeec78"}
{"@t":"2025-02-17T08:22:51.8893815Z","@mt":"Registered with MainDom, localContentDbExists? {LocalContentDbExists}, localMediaDbExists? {LocalMediaDbExists}","LocalContentDbExists":true,"LocalMediaDbExists":true,"SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"be245a42-069a-4183-89ec-73b07fdeec78"}
{"@t":"2025-02-17T08:22:51.8893815Z","@mt":"Creating the content store, localContentDbExists? {LocalContentDbExists}","LocalContentDbExists":true,"SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"be245a42-069a-4183-89ec-73b07fdeec78"}
{"@t":"2025-02-17T08:22:51.8933813Z","@mt":"Creating the media store, localMediaDbExists? {LocalMediaDbExists}","LocalMediaDbExists":true,"SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"be245a42-069a-4183-89ec-73b07fdeec78"}
{"@t":"2025-02-17T08:23:00.9832891Z","@mt":"The culture specified {Culture} was not found in any configured sources for this service","@l":"Warning","Culture":"sk-SK","SourceContext":"Umbraco.Core.Services.Implement.LocalizedTextService","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"WARN ","HttpRequestNumber":1,"HttpRequestId":"be245a42-069a-4183-89ec-73b07fdeec78"}
{"@t":"2025-02-17T08:23:00.9832891Z","@mt":"The culture specified {Culture} was not found in any configured sources for this service","@l":"Warning","Culture":"sk-SK","SourceContext":"Umbraco.Core.Services.Implement.LocalizedTextService","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"WARN ","HttpRequestNumber":1,"HttpRequestId":"be245a42-069a-4183-89ec-73b07fdeec78"}
{"@t":"2025-02-17T08:23:00.9875147Z","@mt":"The culture specified {Culture} was not found in any configured sources for this service","@l":"Warning","Culture":"sk-SK","SourceContext":"Umbraco.Core.Services.Implement.LocalizedTextService","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"WARN ","HttpRequestNumber":1,"HttpRequestId":"be245a42-069a-4183-89ec-73b07fdeec78"}
{"@t":"2025-02-17T08:23:00.9887992Z","@mt":"The culture specified {Culture} was not found in any configured sources for this service","@l":"Warning","Culture":"sk-SK","SourceContext":"Umbraco.Core.Services.Implement.LocalizedTextService","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"WARN ","HttpRequestNumber":1,"HttpRequestId":"be245a42-069a-4183-89ec-73b07fdeec78"}
{"@t":"2025-02-17T08:23:00.9898038Z","@mt":"The culture specified {Culture} was not found in any configured sources for this service","@l":"Warning","Culture":"sk-SK","SourceContext":"Umbraco.Core.Services.Implement.LocalizedTextService","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"WARN ","HttpRequestNumber":1,"HttpRequestId":"be245a42-069a-4183-89ec-73b07fdeec78"}
{"@t":"2025-02-17T08:23:00.9898038Z","@mt":"The culture specified {Culture} was not found in any configured sources for this service","@l":"Warning","Culture":"sk-SK","SourceContext":"Umbraco.Core.Services.Implement.LocalizedTextService","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"WARN ","HttpRequestNumber":1,"HttpRequestId":"be245a42-069a-4183-89ec-73b07fdeec78"}
{"@t":"2025-02-17T08:23:00.9898038Z","@mt":"The culture specified {Culture} was not found in any configured sources for this service","@l":"Warning","Culture":"sk-SK","SourceContext":"Umbraco.Core.Services.Implement.LocalizedTextService","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"WARN ","HttpRequestNumber":1,"HttpRequestId":"be245a42-069a-4183-89ec-73b07fdeec78"}
{"@t":"2025-02-17T08:23:00.9898038Z","@mt":"The culture specified {Culture} was not found in any configured sources for this service","@l":"Warning","Culture":"sk-SK","SourceContext":"Umbraco.Core.Services.Implement.LocalizedTextService","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"WARN ","HttpRequestNumber":1,"HttpRequestId":"be245a42-069a-4183-89ec-73b07fdeec78"}
{"@t":"2025-02-17T08:23:01.0610275Z","@mt":"{StartMessage} [Timing {TimingId}]","StartMessage":"Loading content from local cache file","TimingId":"73e1902","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"be245a42-069a-4183-89ec-73b07fdeec78"}
{"@t":"2025-02-17T08:23:01.2411385Z","@mt":"{EndMessage} ({Duration}ms) [Timing {TimingId}]","EndMessage":"Completed.","Duration":179,"TimingId":"73e1902","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"be245a42-069a-4183-89ec-73b07fdeec78"}
{"@t":"2025-02-17T08:23:01.3677075Z","@mt":"{StartMessage} [Timing {TimingId}]","StartMessage":"Loading media from local cache file","TimingId":"d7da2f5","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"be245a42-069a-4183-89ec-73b07fdeec78"}
{"@t":"2025-02-17T08:23:01.3717281Z","@mt":"{EndMessage} ({Duration}ms) [Timing {TimingId}]","EndMessage":"Completed.","Duration":3,"TimingId":"d7da2f5","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"be245a42-069a-4183-89ec-73b07fdeec78"}
{"@t":"2025-02-17T08:23:01.9457621Z","@mt":"Profiler is VoidProfiler, not profiling (must run debug mode to profile).","SourceContext":"Umbraco.Web.Logging.WebProfilerComponent","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"be245a42-069a-4183-89ec-73b07fdeec78"}
{"@t":"2025-02-17T08:23:02.3507872Z","@mt":"Initializing Umbraco internal event handlers for cache refreshing.","SourceContext":"Umbraco.Web.Cache.DistributedCacheBinderComponent","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"be245a42-069a-4183-89ec-73b07fdeec78"}
{"@t":"2025-02-17T08:23:02.5340647Z","@mt":"Adding examine event handlers for {RegisteredIndexers} index providers.","RegisteredIndexers":3,"SourceContext":"Umbraco.Web.Search.ExamineComponent","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"be245a42-069a-4183-89ec-73b07fdeec78"}
{"@t":"2025-02-17T08:23:02.6123461Z","@mt":"Forcing index {IndexerName} to be unlocked since it was left in a locked state","IndexerName":"InternalIndex","SourceContext":"Umbraco.Examine.ExamineExtensions","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"be245a42-069a-4183-89ec-73b07fdeec78"}
{"@t":"2025-02-17T08:23:02.6147140Z","@mt":"Forcing index {IndexerName} to be unlocked since it was left in a locked state","IndexerName":"MembersIndex","SourceContext":"Umbraco.Examine.ExamineExtensions","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"be245a42-069a-4183-89ec-73b07fdeec78"}
{"@t":"2025-02-17T08:23:02.6157993Z","@mt":"Forcing index {IndexerName} to be unlocked since it was left in a locked state","IndexerName":"ExternalIndex","SourceContext":"Umbraco.Examine.ExamineExtensions","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"be245a42-069a-4183-89ec-73b07fdeec78"}
{"@t":"2025-02-17T08:23:02.6188019Z","@mt":"Starting initialize async background thread.","SourceContext":"Umbraco.Web.Search.BackgroundIndexRebuilder","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"be245a42-069a-4183-89ec-73b07fdeec78"}
{"@t":"2025-02-17T08:23:02.6768050Z","@mt":"Starting '{MigrationName}'...","MigrationName":"UmbracoForms","SourceContext":"Umbraco.Core.Migrations.MigrationPlan","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"be245a42-069a-4183-89ec-73b07fdeec78"}
{"@t":"2025-02-17T08:23:02.6768050Z","@mt":"At {OrigState}","OrigState":"6e692c5d-c670-4c34-af17-28d8dbf0dcd2","SourceContext":"Umbraco.Core.Migrations.MigrationPlan","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"be245a42-069a-4183-89ec-73b07fdeec78"}
{"@t":"2025-02-17T08:23:02.6778050Z","@mt":"Done (pending scope completion).","SourceContext":"Umbraco.Core.Migrations.MigrationPlan","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"be245a42-069a-4183-89ec-73b07fdeec78"}
{"@t":"2025-02-17T08:23:02.7164093Z","@mt":"Initializing Umbraco Forms Events to trigger Cache Refreshers","SourceContext":"Umbraco.Forms.Core.Components.UmbracoFormsComposer","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"be245a42-069a-4183-89ec-73b07fdeec78"}
{"@t":"2025-02-17T08:23:02.7227804Z","@mt":"{EndMessage} ({Duration}ms) [Timing {TimingId}]","EndMessage":"Booted.","Duration":17061,"TimingId":"6cf3b80","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"be245a42-069a-4183-89ec-73b07fdeec78"}
{"@t":"2025-02-17T08:23:03.7519808Z","@mt":"New url {Url} detected, re-discovering application url.","Url":"http://localhost/umbraco","SourceContext":"Umbraco.Core.Sync.ApplicationUrlHelper","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO ","HttpRequestNumber":2,"HttpRequestId":"d67595fa-452e-4514-b7a6-41864116d7c2"}
{"@t":"2025-02-17T08:23:03.7549833Z","@mt":"ApplicationUrl: {UmbracoAppUrl} (UmbracoModule request)","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Core.Sync.ApplicationUrlHelper","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO ","HttpRequestNumber":2,"HttpRequestId":"d67595fa-452e-4514-b7a6-41864116d7c2"}
{"@t":"2025-02-17T08:51:34.4547758Z","@mt":"The culture specified {Culture} was not found in any configured sources for this service","@l":"Warning","Culture":"sk-SK","SourceContext":"Umbraco.Core.Services.Implement.LocalizedTextService","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"WARN "}
{"@t":"2025-02-17T08:51:34.4562869Z","@mt":"The culture specified {Culture} was not found in any configured sources for this service","@l":"Warning","Culture":"sk-SK","SourceContext":"Umbraco.Core.Services.Implement.LocalizedTextService","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"WARN "}
{"@t":"2025-02-17T08:51:34.4929435Z","@mt":"The culture specified {Culture} was not found in any configured sources for this service","@l":"Warning","Culture":"sk-SK","SourceContext":"Umbraco.Core.Services.Implement.LocalizedTextService","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"WARN "}
{"@t":"2025-02-17T08:51:34.5060163Z","@mt":"The culture specified {Culture} was not found in any configured sources for this service","@l":"Warning","Culture":"sk-SK","SourceContext":"Umbraco.Core.Services.Implement.LocalizedTextService","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"WARN "}
{"@t":"2025-02-17T08:51:34.5363476Z","@mt":"The culture specified {Culture} was not found in any configured sources for this service","@l":"Warning","Culture":"sk-SK","SourceContext":"Umbraco.Core.Services.Implement.LocalizedTextService","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"WARN "}
{"@t":"2025-02-17T08:51:34.5363476Z","@mt":"The culture specified {Culture} was not found in any configured sources for this service","@l":"Warning","Culture":"sk-SK","SourceContext":"Umbraco.Core.Services.Implement.LocalizedTextService","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"WARN "}
{"@t":"2025-02-17T08:51:34.5383576Z","@mt":"The culture specified {Culture} was not found in any configured sources for this service","@l":"Warning","Culture":"sk-SK","SourceContext":"Umbraco.Core.Services.Implement.LocalizedTextService","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"WARN "}
{"@t":"2025-02-17T08:51:34.5383576Z","@mt":"The culture specified {Culture} was not found in any configured sources for this service","@l":"Warning","Culture":"sk-SK","SourceContext":"Umbraco.Core.Services.Implement.LocalizedTextService","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"WARN "}
{"@t":"2025-02-17T08:51:34.5403711Z","@mt":"The culture specified {Culture} was not found in any configured sources for this service","@l":"Warning","Culture":"sk-SK","SourceContext":"Umbraco.Core.Services.Implement.LocalizedTextService","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"WARN "}
{"@t":"2025-02-17T08:51:34.5403711Z","@mt":"The culture specified {Culture} was not found in any configured sources for this service","@l":"Warning","Culture":"sk-SK","SourceContext":"Umbraco.Core.Services.Implement.LocalizedTextService","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"WARN "}
{"@t":"2025-02-17T08:51:34.5418878Z","@mt":"The culture specified {Culture} was not found in any configured sources for this service","@l":"Warning","Culture":"sk-SK","SourceContext":"Umbraco.Core.Services.Implement.LocalizedTextService","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"WARN "}
{"@t":"2025-02-17T08:51:34.5418878Z","@mt":"The culture specified {Culture} was not found in any configured sources for this service","@l":"Warning","Culture":"sk-SK","SourceContext":"Umbraco.Core.Services.Implement.LocalizedTextService","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"WARN "}
{"@t":"2025-02-17T08:51:34.5439228Z","@mt":"The culture specified {Culture} was not found in any configured sources for this service","@l":"Warning","Culture":"sk-SK","SourceContext":"Umbraco.Core.Services.Implement.LocalizedTextService","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"WARN "}
{"@t":"2025-02-17T08:51:34.5439228Z","@mt":"The culture specified {Culture} was not found in any configured sources for this service","@l":"Warning","Culture":"sk-SK","SourceContext":"Umbraco.Core.Services.Implement.LocalizedTextService","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"WARN "}
{"@t":"2025-02-17T08:51:34.5479562Z","@mt":"The culture specified {Culture} was not found in any configured sources for this service","@l":"Warning","Culture":"sk-SK","SourceContext":"Umbraco.Core.Services.Implement.LocalizedTextService","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"WARN "}
{"@t":"2025-02-17T08:51:34.5479562Z","@mt":"The culture specified {Culture} was not found in any configured sources for this service","@l":"Warning","Culture":"sk-SK","SourceContext":"Umbraco.Core.Services.Implement.LocalizedTextService","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"WARN "}
{"@t":"2025-02-17T08:51:34.5543894Z","@mt":"The culture specified {Culture} was not found in any configured sources for this service","@l":"Warning","Culture":"sk-SK","SourceContext":"Umbraco.Core.Services.Implement.LocalizedTextService","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"WARN "}
{"@t":"2025-02-17T08:51:34.5564032Z","@mt":"The culture specified {Culture} was not found in any configured sources for this service","@l":"Warning","Culture":"sk-SK","SourceContext":"Umbraco.Core.Services.Implement.LocalizedTextService","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"WARN "}
{"@t":"2025-02-17T08:51:34.5581541Z","@mt":"The culture specified {Culture} was not found in any configured sources for this service","@l":"Warning","Culture":"sk-SK","SourceContext":"Umbraco.Core.Services.Implement.LocalizedTextService","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"WARN "}
{"@t":"2025-02-17T08:51:38.6059635Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in d:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":31,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"ERROR"}
{"@t":"2025-02-17T08:51:38.6573692Z","@mt":"The culture specified {Culture} was not found in any configured sources for this service","@l":"Warning","Culture":"sk-SK","SourceContext":"Umbraco.Core.Services.Implement.LocalizedTextService","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"WARN "}
{"@t":"2025-02-17T08:51:38.6573692Z","@mt":"The culture specified {Culture} was not found in any configured sources for this service","@l":"Warning","Culture":"sk-SK","SourceContext":"Umbraco.Core.Services.Implement.LocalizedTextService","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"WARN "}
{"@t":"2025-02-17T08:51:38.6573692Z","@mt":"The culture specified {Culture} was not found in any configured sources for this service","@l":"Warning","Culture":"sk-SK","SourceContext":"Umbraco.Core.Services.Implement.LocalizedTextService","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"WARN "}
{"@t":"2025-02-17T08:51:42.7142852Z","@mt":"The culture specified {Culture} was not found in any configured sources for this service","@l":"Warning","Culture":"sk-SK","SourceContext":"Umbraco.Core.Services.Implement.LocalizedTextService","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"WARN "}
{"@t":"2025-02-17T08:51:42.7142852Z","@mt":"The culture specified {Culture} was not found in any configured sources for this service","@l":"Warning","Culture":"sk-SK","SourceContext":"Umbraco.Core.Services.Implement.LocalizedTextService","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"WARN "}
{"@t":"2025-02-17T08:51:42.7142852Z","@mt":"The culture specified {Culture} was not found in any configured sources for this service","@l":"Warning","Culture":"sk-SK","SourceContext":"Umbraco.Core.Services.Implement.LocalizedTextService","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"WARN "}
{"@t":"2025-02-17T08:51:46.7658203Z","@mt":"The culture specified {Culture} was not found in any configured sources for this service","@l":"Warning","Culture":"sk-SK","SourceContext":"Umbraco.Core.Services.Implement.LocalizedTextService","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"WARN "}
{"@t":"2025-02-17T08:51:46.7658203Z","@mt":"The culture specified {Culture} was not found in any configured sources for this service","@l":"Warning","Culture":"sk-SK","SourceContext":"Umbraco.Core.Services.Implement.LocalizedTextService","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"WARN "}
{"@t":"2025-02-17T08:51:46.7658203Z","@mt":"The culture specified {Culture} was not found in any configured sources for this service","@l":"Warning","Culture":"sk-SK","SourceContext":"Umbraco.Core.Services.Implement.LocalizedTextService","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"WARN "}
{"@t":"2025-02-17T08:51:50.8292650Z","@mt":"The culture specified {Culture} was not found in any configured sources for this service","@l":"Warning","Culture":"sk-SK","SourceContext":"Umbraco.Core.Services.Implement.LocalizedTextService","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"WARN "}
{"@t":"2025-02-17T08:51:54.8950773Z","@mt":"The culture specified {Culture} was not found in any configured sources for this service","@l":"Warning","Culture":"sk-SK","SourceContext":"Umbraco.Core.Services.Implement.LocalizedTextService","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"WARN "}
{"@t":"2025-02-17T08:51:54.8950773Z","@mt":"The culture specified {Culture} was not found in any configured sources for this service","@l":"Warning","Culture":"sk-SK","SourceContext":"Umbraco.Core.Services.Implement.LocalizedTextService","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"WARN "}
{"@t":"2025-02-17T08:51:54.8950773Z","@mt":"The culture specified {Culture} was not found in any configured sources for this service","@l":"Warning","Culture":"sk-SK","SourceContext":"Umbraco.Core.Services.Implement.LocalizedTextService","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"WARN "}
{"@t":"2025-02-17T08:51:54.9066050Z","@mt":"The culture specified {Culture} was not found in any configured sources for this service","@l":"Warning","Culture":"sk-SK","SourceContext":"Umbraco.Core.Services.Implement.LocalizedTextService","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"WARN "}
{"@t":"2025-02-17T08:51:54.9126476Z","@mt":"The culture specified {Culture} was not found in any configured sources for this service","@l":"Warning","Culture":"sk-SK","SourceContext":"Umbraco.Core.Services.Implement.LocalizedTextService","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"WARN "}
{"@t":"2025-02-17T08:51:58.9842333Z","@mt":"The culture specified {Culture} was not found in any configured sources for this service","@l":"Warning","Culture":"sk-SK","SourceContext":"Umbraco.Core.Services.Implement.LocalizedTextService","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"WARN "}
{"@t":"2025-02-17T08:51:59.5037551Z","@mt":"Scheduled health check results:","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO "}
{"@t":"2025-02-17T08:51:59.5037551Z","@mt":"Checks for '{HealthCheckName}' all completed successfully.","HealthCheckName":"Folder & File Permissions","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO "}
{"@t":"2025-02-17T08:51:59.5037551Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Folder & File Permissions","HealthCheckResult":"Success","HealthCheckMessage":"[folderPermissionsCheckMessage]","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO "}
{"@t":"2025-02-17T08:51:59.5037551Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Folder & File Permissions","HealthCheckResult":"Success","HealthCheckMessage":"[filePermissionsCheckMessage]","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO "}
{"@t":"2025-02-17T08:51:59.5037551Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"SMTP Settings","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"WARN "}
{"@t":"2025-02-17T08:51:59.5037551Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"SMTP Settings","HealthCheckResult":"Error","HealthCheckMessage":"[smtpMailSettingsHostNotConfigured]","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO "}
{"@t":"2025-02-17T08:51:59.5037551Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"Macro errors","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"WARN "}
{"@t":"2025-02-17T08:51:59.5037551Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Macro errors","HealthCheckResult":"Error","HealthCheckMessage":"[macroErrorModeCheckErrorMessage]","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO "}
{"@t":"2025-02-17T08:51:59.5037551Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"Notification Email Settings","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"WARN "}
{"@t":"2025-02-17T08:51:59.5037551Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Notification Email Settings","HealthCheckResult":"Error","HealthCheckMessage":"[notificationEmailsCheckErrorMessage]","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO "}
{"@t":"2025-02-17T08:51:59.5037551Z","@mt":"Checks for '{HealthCheckName}' all completed successfully.","HealthCheckName":"Try Skip IIS Custom Errors","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO "}
{"@t":"2025-02-17T08:51:59.5037551Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Try Skip IIS Custom Errors","HealthCheckResult":"Success","HealthCheckMessage":"[trySkipIisCustomErrorsCheckSuccessMessage]","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO "}
{"@t":"2025-02-17T08:51:59.5037551Z","@mt":"Checks for '{HealthCheckName}' all completed successfully.","HealthCheckName":"Custom Errors","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO "}
{"@t":"2025-02-17T08:51:59.5037551Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Custom Errors","HealthCheckResult":"Success","HealthCheckMessage":"[customErrorsCheckSuccessMessage]","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO "}
{"@t":"2025-02-17T08:51:59.5037551Z","@mt":"Checks for '{HealthCheckName}' all completed successfully.","HealthCheckName":"Trace Mode","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO "}
{"@t":"2025-02-17T08:51:59.5037551Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Trace Mode","HealthCheckResult":"Success","HealthCheckMessage":"[traceModeCheckSuccessMessage]","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO "}
{"@t":"2025-02-17T08:51:59.5037551Z","@mt":"Checks for '{HealthCheckName}' all completed successfully.","HealthCheckName":"Debug Compilation Mode","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO "}
{"@t":"2025-02-17T08:51:59.5037551Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Debug Compilation Mode","HealthCheckResult":"Success","HealthCheckMessage":"[compilationDebugCheckSuccessMessage]","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO "}
{"@t":"2025-02-17T08:51:59.5037551Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"Cross-site scripting Protection (X-XSS-Protection header)","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"WARN "}
{"@t":"2025-02-17T08:51:59.5037551Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Cross-site scripting Protection (X-XSS-Protection header)","HealthCheckResult":"Error","HealthCheckMessage":"[healthCheckInvalidUrl]","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO "}
{"@t":"2025-02-17T08:51:59.5037551Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"Cookie hijacking and protocol downgrade attacks Protection (Strict-Transport-Security Header (HSTS))","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"WARN "}
{"@t":"2025-02-17T08:51:59.5037551Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Cookie hijacking and protocol downgrade attacks Protection (Strict-Transport-Security Header (HSTS))","HealthCheckResult":"Error","HealthCheckMessage":"[healthCheckInvalidUrl]","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO "}
{"@t":"2025-02-17T08:51:59.5037551Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"Content/MIME Sniffing Protection","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"WARN "}
{"@t":"2025-02-17T08:51:59.5037551Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Content/MIME Sniffing Protection","HealthCheckResult":"Error","HealthCheckMessage":"[healthCheckInvalidUrl]","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO "}
{"@t":"2025-02-17T08:51:59.5037551Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"Excessive Headers","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"WARN "}
{"@t":"2025-02-17T08:51:59.5037551Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Excessive Headers","HealthCheckResult":"Warning","HealthCheckMessage":"[httpsCheckInvalidUrl]","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO "}
{"@t":"2025-02-17T08:51:59.5037551Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"Click-Jacking Protection","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"WARN "}
{"@t":"2025-02-17T08:51:59.5037551Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Click-Jacking Protection","HealthCheckResult":"Error","HealthCheckMessage":"[healthCheckInvalidUrl]","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO "}
{"@t":"2025-02-17T08:51:59.5037551Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"HTTPS Configuration","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"WARN "}
{"@t":"2025-02-17T08:51:59.5037551Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"HTTPS Configuration","HealthCheckResult":"Error","HealthCheckMessage":"[httpsCheckIsCurrentSchemeHttps]","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO "}
{"@t":"2025-02-17T08:51:59.5037551Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"HTTPS Configuration","HealthCheckResult":"Info","HealthCheckMessage":"[httpsCheckConfigurationRectifyNotPossible]","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO "}
{"@t":"2025-02-17T08:51:59.5037551Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"HTTPS Configuration","HealthCheckResult":"Error","HealthCheckMessage":"[healthCheckInvalidUrl]","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO "}
{"@t":"2025-02-17T08:51:59.5057662Z","@mt":"Checks for '{HealthCheckName}' all completed successfully.","HealthCheckName":"Database Integrity","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO "}
{"@t":"2025-02-17T08:51:59.5057662Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Database Integrity","HealthCheckResult":"Success","HealthCheckMessage":"Primary key on table 'UFRecordDataBit' found.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO "}
{"@t":"2025-02-17T08:51:59.5057662Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Database Integrity","HealthCheckResult":"Success","HealthCheckMessage":"Primary key on table 'UFRecordDataDateTime' found.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO "}
{"@t":"2025-02-17T08:51:59.5057662Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Database Integrity","HealthCheckResult":"Success","HealthCheckMessage":"Primary key on table 'UFRecordDataInteger' found.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO "}
{"@t":"2025-02-17T08:51:59.5057662Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Database Integrity","HealthCheckResult":"Success","HealthCheckMessage":"Primary key on table 'UFRecordDataLongString' found.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO "}
{"@t":"2025-02-17T08:51:59.5057662Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Database Integrity","HealthCheckResult":"Success","HealthCheckMessage":"Primary key on table 'UFRecordDataString' found.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO "}
{"@t":"2025-02-17T08:51:59.5057662Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Database Integrity","HealthCheckResult":"Success","HealthCheckMessage":"Primary key on table 'UFUserSecurity' found.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO "}
{"@t":"2025-02-17T08:51:59.5057662Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Database Integrity","HealthCheckResult":"Success","HealthCheckMessage":"Primary key on table 'UFUserFormSecurity' found.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO "}
{"@t":"2025-02-17T08:51:59.5057662Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Database Integrity","HealthCheckResult":"Success","HealthCheckMessage":"Foreign key on table 'UFRecordFields', field 'Record', referencing table 'UFRecords' found.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO "}
{"@t":"2025-02-17T08:51:59.5057662Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Database Integrity","HealthCheckResult":"Success","HealthCheckMessage":"Foreign key on table 'UFRecordDataBit', field 'Key', referencing table 'UFRecordFields' found.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO "}
{"@t":"2025-02-17T08:51:59.5057662Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Database Integrity","HealthCheckResult":"Success","HealthCheckMessage":"Foreign key on table 'UFRecordDataDateTime', field 'Key', referencing table 'UFRecordFields' found.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO "}
{"@t":"2025-02-17T08:51:59.5057662Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Database Integrity","HealthCheckResult":"Success","HealthCheckMessage":"Foreign key on table 'UFRecordDataInteger', field 'Key', referencing table 'UFRecordFields' found.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO "}
{"@t":"2025-02-17T08:51:59.5057662Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Database Integrity","HealthCheckResult":"Success","HealthCheckMessage":"Foreign key on table 'UFRecordDataLongString', field 'Key', referencing table 'UFRecordFields' found.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO "}
{"@t":"2025-02-17T08:51:59.5057662Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Database Integrity","HealthCheckResult":"Success","HealthCheckMessage":"Foreign key on table 'UFRecordDataString', field 'Key', referencing table 'UFRecordFields' found.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO "}
{"@t":"2025-02-17T08:51:59.5057662Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Database Integrity","HealthCheckResult":"Success","HealthCheckMessage":"Unique constraint on table 'UFUserFormSecurity', columns 'User, Form' found.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO "}
{"@t":"2025-02-17T08:51:59.5057662Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Database Integrity","HealthCheckResult":"Success","HealthCheckMessage":"Index on table 'UFRecordDataBit', column 'Key' found.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO "}
{"@t":"2025-02-17T08:51:59.5057662Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Database Integrity","HealthCheckResult":"Success","HealthCheckMessage":"Index on table 'UFRecordDataDateTime', column 'Key' found.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO "}
{"@t":"2025-02-17T08:51:59.5057662Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Database Integrity","HealthCheckResult":"Success","HealthCheckMessage":"Index on table 'UFRecordDataInteger', column 'Key' found.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO "}
{"@t":"2025-02-17T08:51:59.5057662Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Database Integrity","HealthCheckResult":"Success","HealthCheckMessage":"Index on table 'UFRecordDataLongString', column 'Key' found.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO "}
{"@t":"2025-02-17T08:51:59.5057662Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Database Integrity","HealthCheckResult":"Success","HealthCheckMessage":"Index on table 'UFRecordDataString', column 'Key' found.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO "}
{"@t":"2025-02-17T08:56:42.6780564Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in d:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":33,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"ERROR"}
{"@t":"2025-02-17T12:49:49.1948636Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in d:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":32,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"ERROR"}
{"@t":"2025-02-17T12:54:53.2489764Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in d:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":31,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"ERROR"}
{"@t":"2025-02-17T12:59:57.2959555Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in d:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":42,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"ERROR"}
{"@t":"2025-02-17T13:01:13.0831536Z","@mt":"[{EventType}] Event Id: {EventId}, State: {State}","EventType":"Information","EventId":0,"State":"Login attempt succeeded for username svrcekigor@azet.sk from IP address ::1","SourceContext":"Umbraco.Core.Logging.OwinLogger","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":21,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO ","HttpRequestNumber":3,"HttpRequestId":"2e9bcd92-00cf-421e-aef2-ed36f1fffcf3"}
{"@t":"2025-02-17T13:01:13.0831536Z","@mt":"[{EventType}] Event Id: {EventId}, State: {State}","EventType":"Information","EventId":0,"State":"User: svrcekigor@azet.sk logged in from IP address ::1","SourceContext":"Umbraco.Core.Logging.OwinLogger","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":21,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO ","HttpRequestNumber":3,"HttpRequestId":"2e9bcd92-00cf-421e-aef2-ed36f1fffcf3"}
{"@t":"2025-02-17T13:05:01.3654504Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in d:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":41,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"ERROR"}
{"@t":"2025-02-17T13:09:23.5828845Z","@mt":"Document {ContentName} (id={ContentId}) has been published.","ContentName":"About","ContentId":1071,"SourceContext":"Umbraco.Core.Services.Implement.ContentService","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":29,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO ","HttpRequestNumber":4,"HttpRequestId":"e907328c-7a13-4b11-9cbf-c41750154705"}
{"@t":"2025-02-17T13:10:05.4715025Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in d:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":32,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"ERROR"}
{"@t":"2025-02-17T13:15:09.5336644Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in d:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":42,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"ERROR"}
{"@t":"2025-02-17T13:20:13.5973006Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in d:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":47,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"ERROR"}
{"@t":"2025-02-17T13:25:17.6414032Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in d:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":31,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"ERROR"}
{"@t":"2025-02-17T13:30:21.7338370Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in d:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":46,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"ERROR"}
{"@t":"2025-02-17T13:35:25.8483616Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in d:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":42,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"ERROR"}
{"@t":"2025-02-17T13:38:49.5490036Z","@mt":"[{EventType}] Event Id: {EventId}, State: {State}","EventType":"Information","EventId":0,"State":"Login attempt succeeded for username svrcekigor@azet.sk from IP address ::1","SourceContext":"Umbraco.Core.Logging.OwinLogger","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":36,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO ","HttpRequestNumber":5,"HttpRequestId":"38d48afc-771a-4986-a57a-6a59ba98badb"}
{"@t":"2025-02-17T13:38:49.5490036Z","@mt":"[{EventType}] Event Id: {EventId}, State: {State}","EventType":"Information","EventId":0,"State":"User: svrcekigor@azet.sk logged in from IP address ::1","SourceContext":"Umbraco.Core.Logging.OwinLogger","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":36,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO ","HttpRequestNumber":5,"HttpRequestId":"38d48afc-771a-4986-a57a-6a59ba98badb"}
{"@t":"2025-02-17T13:40:29.9244162Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in d:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":44,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"ERROR"}
{"@t":"2025-02-17T13:45:33.9759231Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in d:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":6,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"ERROR"}
{"@t":"2025-02-17T13:50:38.0519157Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in d:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":52,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"ERROR"}
{"@t":"2025-02-17T14:08:19.5897052Z","@mt":"[{EventType}] Event Id: {EventId}, State: {State}","EventType":"Information","EventId":0,"State":"User logged will be logged out due to timeout: svrcekigor@azet.sk, IP Address: ::1","SourceContext":"Umbraco.Core.Logging.OwinLogger","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":42,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO ","HttpRequestNumber":6,"HttpRequestId":"23596041-94a8-45d1-a958-e863dd9fe63d"}
{"@t":"2025-02-17T14:08:28.7634253Z","@mt":"[{EventType}] Event Id: {EventId}, State: {State}","EventType":"Information","EventId":0,"State":"Login attempt succeeded for username svrcekigor@azet.sk from IP address ::1","SourceContext":"Umbraco.Core.Logging.OwinLogger","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":43,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO ","HttpRequestNumber":7,"HttpRequestId":"2faa80cd-9273-4641-94e2-580bf473822c"}
{"@t":"2025-02-17T14:08:28.7634253Z","@mt":"[{EventType}] Event Id: {EventId}, State: {State}","EventType":"Information","EventId":0,"State":"User: svrcekigor@azet.sk logged in from IP address ::1","SourceContext":"Umbraco.Core.Logging.OwinLogger","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":43,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO ","HttpRequestNumber":7,"HttpRequestId":"2faa80cd-9273-4641-94e2-580bf473822c"}
{"@t":"2025-02-17T14:08:44.4696257Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in d:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":57,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"ERROR"}
{"@t":"2025-02-17T14:13:48.5331265Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in d:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":40,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"ERROR"}
{"@t":"2025-02-17T14:18:52.6311901Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in d:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":62,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"ERROR"}
{"@t":"2025-02-17T14:23:56.7286153Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in d:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":71,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"ERROR"}
{"@t":"2025-02-17T14:29:00.8162933Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in d:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":63,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"ERROR"}
{"@t":"2025-02-17T14:32:36.3895499Z","@mt":"Document {ContentName} (id={ContentId}) has been published.","ContentName":"About","ContentId":1071,"SourceContext":"Umbraco.Core.Services.Implement.ContentService","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":64,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO ","HttpRequestNumber":8,"HttpRequestId":"e6061b25-4cbc-4813-970c-c8ab2c287a2b"}
{"@t":"2025-02-17T14:34:04.8981440Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in d:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":91,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"ERROR"}
{"@t":"2025-02-17T20:21:50.2730460Z","@mt":"Exception ({InstanceId}).","@l":"Error","@x":"System.Data.SqlClient.SqlException (0x80131904): Time-out occurred while waiting for buffer latch type 2 for page (1:448), database ID 22. (LATCH 0x0000024D73212858: Class: BUFFER KP: 0 SH: 0 UP: 0 EX: 0 DT: 0 Sublatch: 0 HasWaiters: 0 Task: 0x0000000000000000 AnyReleasor: 0)\r\n   at System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)\r\n   at System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)\r\n   at System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)\r\n   at System.Data.SqlClient.SqlCommand.FinishExecuteReader(SqlDataReader ds, RunBehavior runBehavior, String resetOptionsString, Boolean isInternal, Boolean forDescribeParameterEncryption, Boolean shouldCacheForAlwaysEncrypted)\r\n   at System.Data.SqlClient.SqlCommand.RunExecuteReaderTds(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, Boolean async, Int32 timeout, Task& task, Boolean asyncWrite, Boolean inRetry, SqlDataReader ds, Boolean describeParameterEncryptionRequest)\r\n   at System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method, TaskCompletionSource`1 completion, Int32 timeout, Task& task, Boolean& usedCache, Boolean asyncWrite, Boolean inRetry)\r\n   at System.Data.SqlClient.SqlCommand.InternalExecuteNonQuery(TaskCompletionSource`1 completion, String methodName, Boolean sendToPipe, Int32 timeout, Boolean& usedCache, Boolean asyncWrite, Boolean inRetry)\r\n   at System.Data.SqlClient.SqlCommand.ExecuteNonQuery()\r\n   at Umbraco.Core.Persistence.FaultHandling.RetryPolicy.ExecuteAction[TResult](Func`1 func) in d:\\a\\1\\s\\src\\Umbraco.Core\\Persistence\\FaultHandling\\RetryPolicy.cs:line 172\r\n   at NPoco.Database.ExecuteNonQueryHelper(DbCommand cmd)\r\n   at NPoco.Database.Execute(String sql, CommandType commandType, Object[] args)\r\nClientConnectionId:0f620d00-2dbd-4faa-bf7d-450c8f5d3965\r\nError Number:845,State:1,Class:17","InstanceId":"bb31371c","SourceContext":"Umbraco.Core.Persistence.UmbracoDatabase","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":46,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"ERROR"}
{"@t":"2025-02-17T20:21:52.0822968Z","@mt":"Failed.","@l":"Error","@x":"System.Data.SqlClient.SqlException (0x80131904): Time-out occurred while waiting for buffer latch type 2 for page (1:448), database ID 22. (LATCH 0x0000024D73212858: Class: BUFFER KP: 0 SH: 0 UP: 0 EX: 0 DT: 0 Sublatch: 0 HasWaiters: 0 Task: 0x0000000000000000 AnyReleasor: 0)\r\n   at System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)\r\n   at System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)\r\n   at System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)\r\n   at System.Data.SqlClient.SqlCommand.FinishExecuteReader(SqlDataReader ds, RunBehavior runBehavior, String resetOptionsString, Boolean isInternal, Boolean forDescribeParameterEncryption, Boolean shouldCacheForAlwaysEncrypted)\r\n   at System.Data.SqlClient.SqlCommand.RunExecuteReaderTds(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, Boolean async, Int32 timeout, Task& task, Boolean asyncWrite, Boolean inRetry, SqlDataReader ds, Boolean describeParameterEncryptionRequest)\r\n   at System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method, TaskCompletionSource`1 completion, Int32 timeout, Task& task, Boolean& usedCache, Boolean asyncWrite, Boolean inRetry)\r\n   at System.Data.SqlClient.SqlCommand.InternalExecuteNonQuery(TaskCompletionSource`1 completion, String methodName, Boolean sendToPipe, Int32 timeout, Boolean& usedCache, Boolean asyncWrite, Boolean inRetry)\r\n   at System.Data.SqlClient.SqlCommand.ExecuteNonQuery()\r\n   at Umbraco.Core.Persistence.FaultHandling.RetryPolicy.ExecuteAction[TResult](Func`1 func) in d:\\a\\1\\s\\src\\Umbraco.Core\\Persistence\\FaultHandling\\RetryPolicy.cs:line 172\r\n   at NPoco.Database.ExecuteNonQueryHelper(DbCommand cmd)\r\n   at NPoco.Database.Execute(String sql, CommandType commandType, Object[] args)\r\n   at Umbraco.Core.Persistence.SqlSyntax.SqlServerSyntaxProvider.WriteLock(IDatabase db, TimeSpan timeout, Int32[] lockIds) in d:\\a\\1\\s\\src\\Umbraco.Core\\Persistence\\SqlSyntax\\SqlServerSyntaxProvider.cs:line 269\r\n   at Umbraco.Core.Services.Implement.ContentService.<PerformScheduledPublishInternal>d__59.MoveNext() in d:\\a\\1\\s\\src\\Umbraco.Core\\Services\\Implement\\ContentService.cs:line 1376\r\n   at System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)\r\n   at System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)\r\n   at Umbraco.Core.Services.Implement.ContentService.PerformScheduledPublish(DateTime date) in d:\\a\\1\\s\\src\\Umbraco.Core\\Services\\Implement\\ContentService.cs:line 1363\r\n   at Umbraco.Web.Scheduling.ScheduledPublishing.PerformRun() in d:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\ScheduledPublishing.cs:line 72\r\nClientConnectionId:0f620d00-2dbd-4faa-bf7d-450c8f5d3965\r\nError Number:845,State:1,Class:17","SourceContext":"Umbraco.Web.Scheduling.ScheduledPublishing","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":46,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"ERROR"}
{"@t":"2025-02-17T20:24:54.8843877Z","@mt":"[{EventType}] Event Id: {EventId}, State: {State}","EventType":"Information","EventId":0,"State":"Login attempt succeeded for username svrcekigor@azet.sk from IP address ::1","SourceContext":"Umbraco.Core.Logging.OwinLogger","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":64,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO ","HttpRequestNumber":9,"HttpRequestId":"6f144b0c-db23-44eb-9d59-376b16920ab9"}
{"@t":"2025-02-17T20:24:54.8843877Z","@mt":"[{EventType}] Event Id: {EventId}, State: {State}","EventType":"Information","EventId":0,"State":"User: svrcekigor@azet.sk logged in from IP address ::1","SourceContext":"Umbraco.Core.Logging.OwinLogger","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":64,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO ","HttpRequestNumber":9,"HttpRequestId":"6f144b0c-db23-44eb-9d59-376b16920ab9"}
{"@t":"2025-02-17T20:25:56.5152427Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in d:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":57,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"ERROR"}
{"@t":"2025-02-17T20:31:00.5972457Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in d:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":94,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"ERROR"}
{"@t":"2025-02-17T20:36:04.6420362Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in d:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":92,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"ERROR"}
{"@t":"2025-02-17T20:41:08.7449158Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in d:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":92,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"ERROR"}
{"@t":"2025-02-17T20:46:12.8229124Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in d:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":40,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"ERROR"}
{"@t":"2025-02-17T20:51:16.9363244Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in d:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":63,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"ERROR"}
{"@t":"2025-02-17T20:56:21.0172380Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in d:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":62,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"ERROR"}
{"@t":"2025-02-17T21:01:25.0813350Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in d:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":96,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"ERROR"}
{"@t":"2025-02-17T21:05:52.5541870Z","@mt":"Document {ContentName} (id={ContentId}) has been published.","ContentName":"About","ContentId":1071,"SourceContext":"Umbraco.Core.Services.Implement.ContentService","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":74,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO ","HttpRequestNumber":10,"HttpRequestId":"320d042f-e6aa-4ac3-918c-76ef73ea20a8"}
{"@t":"2025-02-17T21:06:29.1633730Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in d:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":95,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"ERROR"}
{"@t":"2025-02-17T21:11:33.2652562Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in d:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":54,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"ERROR"}
{"@t":"2025-02-17T21:16:37.3628100Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in d:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":57,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"ERROR"}
{"@t":"2025-02-17T21:21:41.4561572Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in d:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":31,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"ERROR"}
{"@t":"2025-02-17T21:26:45.5237396Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in d:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":43,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"ERROR"}
{"@t":"2025-02-17T21:28:45.2537508Z","@mt":"Document {ContentName} (id={ContentId}) has been published.","ContentName":"About","ContentId":1071,"SourceContext":"Umbraco.Core.Services.Implement.ContentService","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":80,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO ","HttpRequestNumber":11,"HttpRequestId":"456ee466-b7a3-4102-94ec-81a96ea70d0f"}
{"@t":"2025-02-17T21:29:34.9285444Z","@mt":"Document {ContentName} (id={ContentId}) has been published.","ContentName":"About","ContentId":1071,"SourceContext":"Umbraco.Core.Services.Implement.ContentService","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":59,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO ","HttpRequestNumber":12,"HttpRequestId":"f862e5bd-7fb0-436a-a613-9e29f72d8501"}
{"@t":"2025-02-17T21:31:49.5999541Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in d:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":58,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"ERROR"}
{"@t":"2025-02-17T21:36:53.6840251Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in d:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":58,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"ERROR"}
{"@t":"2025-02-17T21:39:01.3514869Z","@mt":"Document {ContentName} (id={ContentId}) has been published.","ContentName":"Contact","ContentId":1072,"SourceContext":"Umbraco.Core.Services.Implement.ContentService","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":57,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"INFO ","HttpRequestNumber":13,"HttpRequestId":"e588e75a-59de-482e-826c-b5d7e0953f46"}
{"@t":"2025-02-17T21:41:57.7443713Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in d:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":55,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"ERROR"}
{"@t":"2025-02-17T21:47:01.7862061Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in d:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":27,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"ERROR"}
{"@t":"2025-02-17T21:52:05.8389311Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in d:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":56,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"ERROR"}
{"@t":"2025-02-17T21:57:09.8932284Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in d:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":14,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"ERROR"}
{"@t":"2025-02-17T22:04:39.7955390Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in d:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":61,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"ERROR"}
{"@t":"2025-02-17T22:09:43.8748172Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in d:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":10608,"ProcessName":"iisexpress","ThreadId":55,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-7JFF6SL","Log4NetLevel":"ERROR"}
